src/StakeNFT.sol

```diff
@@ -277,54 +267,57 @@ contract StakeNFT is ERC721, MagicValue, Admin, Governance, CircuitBreaker, Atom
     function _mintNFT(address to_, uint256 amount_) internal returns(uint256 tokenID) {
         // amount must be less than maxUInt32 - this is to allow struct packing
```

Comment:
Can this comment be updated? We are not using uint32 at this point.

Reply:
Good catch! Fixed!